react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -105,22 +139,46 @@ class ToolbarButton extends Component {
      * @returns {ReactElement}
      */
     render(): ReactElement<*> {
-        const { button } = this.props;
-        const popups = button.popups || [];
+        const { button, t, tooltipPosition } = this.props;
+        const popupConfig = this._getPopupDisplayConfiguration();
 
         const props = {
             ...this.props,
+            onClick: this._onClick,
             createRefToButton: this._createRefToButton
         };
 
         return (
-            <StatelessToolbarButton { ...props }>
-                { this._renderPopups(popups) }
-            </StatelessToolbarButton>
+            <AKInlineDialog
```

Comment:
We may want to add a comment here saying that this for us is a popup. We may even want to add a wrapper component at some point that adds some features we'd like to use everywhere and call it a popup.